ras_party/views/share_survey_view.py

```diff
@@ -150,5 +150,15 @@ def confirm_pending_shares(batch_no):
     Confirms pending share survey
     :param batch_no
     """
-    accept_share_survey(batch_no)
+    confirm_share_survey(batch_no)
     return make_response(jsonify(), 201)
+
+
+@share_survey_view.route('/share-survey/<batch_no>', methods=['GET'])
+def get_pending_share_with_batch_no(batch_no):
+    """
+    Confirms pending share survey
+    :param batch_no
+    """
+    response = get_share_survey_by_batch_number(batch_no)
+    return make_response(jsonify(response), 200)
```

Comment:
I'm pretty sure this can be replaced with 
```python
return jsonify(response)
```
as you're not doing anything fancy with the status

Reply:
I am sure you are right. But with respect to other code keeping it consistent can we ignore it for now.

Reply:
yeah sure